src/Config/Environment.php

```diff
@@ -26,7 +26,7 @@ class Environment
     /**
      * The environment variable for controlling the directory nesting level for error reporting
      */
-    private const ENV_MAGE_ERROR_REPORT_DIR_NESTING_LEVEL = 'MAGE_ERROR_REPORT_DIR_NESTING_LEVEL';
+    public const ENV_MAGE_ERROR_REPORT_DIR_NESTING_LEVEL = 'MAGE_ERROR_REPORT_DIR_NESTING_LEVEL';
```

Comment:
Better to keep this constant private and in Unit test use it string value. 
Because now the unit test will not fail if someone changes the constant.

Reply:
We don't test the value of constant, but the functionality which uses it

Reply:
In some cases, we should test the value of constants as well. As this value can't be changed.

Reply:
Ok, it makes sense for this case then. Changed.